src/Spryker/Zed/EventBehavior/Persistence/EventBehaviorQueryContainer.php

```diff
@@ -65,6 +65,24 @@ public function queryLatestEntityChange(DateTime $date)
         return $query;
     }
 
+    /**
+     * {@inheritDoc}
+     *
+     * @api
+     *
+     * @param array $keys
+     *
+     * @return \Orm\Zed\EventBehavior\Persistence\SpyEventBehaviorEntityChangeQuery
+     */
+    public function queryEntityByKeys(array $keys)
+    {
+        $query = $this->getFactory()
+            ->createEventBehaviorEntityChangeQuery()
+            ->filterByPrimaryKeys($keys);
+
+        return $query;
```

Comment:
Missed return type.
Also I think we can avoid of creating $query variable

Reply:
@olhalivitchuk  done